app/scripts/controllers/dashboard.coffee

```diff
@@ -117,6 +121,22 @@ angular.module('edudashAppCtrl').controller 'DashboardCtrl', (
                 resolve maxRatio
               ), reject
 
+        watchCompute 'gpaComputedMax',
+          dependencies: ['allSchools']
+          waitForPromise: true
+          computer: ([allSchools]) -> $q (resolve, reject) ->
+            MIN = 5
+            unless allSchools?
+              resolve MIN
+            else
+              allSchools.then ((schools) ->
+                ratios = schools
```

Comment:
Alternatively, we could make a function that takes `(allSchoolsPromise, prop, defaultMax) ->` in `utils` and use it for both `ptratioComputedMax` and `gpaComputedMax` (and a few unit tests wouldn't hurt).